api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/ClaimService.java

```diff
@@ -123,6 +124,7 @@ private void sendNewCardMessage(Claim claim, EligibilityAndEntitlementDecision d
         messageQueueDAO.sendMessage(payload, CREATE_NEW_CARD);
     }
 
+    @SuppressFBWarnings("SECMD5") // ignore spotbugs warning about md5 as we're not using it for encryption
```

Comment:
Can we get a bit more info on what was causing this and why?

Reply:
It's for hashing the device fingerprint, so we can do comparisons against the hash rather than the whole thing.

Reply:
Rather than a comment, this annotation takes in a justification value you can use instead.